openslides/agenda/projector.py

```diff
@@ -38,6 +38,11 @@ def get_requirements(self, config_entry):
         yield ProjectorRequirement(
             view_class=ItemViewSet,
             view_action='list')
+        for speaker in self.speakers.all():
```

Comment:
Ok. But care of my objection in the other pull request. Maybe it is not worth spending much time on that approach.